packages/bitcore-lib-cash/lib/crypto/signature.js

```diff
@@ -30,7 +30,7 @@ Signature.prototype.set = function(obj) {
   this.i = typeof obj.i !== 'undefined' ? obj.i : this.i; //public key recovery parameter in range [0, 3]
   this.compressed = typeof obj.compressed !== 'undefined' ?
     obj.compressed : this.compressed; // whether the recovered pubkey is compressed
-  this.isSchnorr = obj.isSchnorr;
+  this.isSchnorr = obj.isSchnorr || this.isSchnorr || undefined;
```

Comment:
Thanks for the review! I actually made this change precisely because the behavior you describe (`this.isSchnorr` being overwritten to `undefined` if someone calls .set without `obj.isSchnorr` passed in) is actually what is currently happening in the latest master, and this line fixes it.

Reply:
You're right -- sorry I read it backwards. This is good.